test/lib/ufe/testParentCmd.py

```diff
@@ -85,6 +85,17 @@ def testParentRelative(self):
             [shapeSegment, usdUtils.createUfePathSegment("/pCylinder1")])
         cylinderItem = ufe.Hierarchy.createItem(cylinderPath)
 
+        # get the USD stage
+        stage = mayaUsd.ufe.getStage(str(shapeSegment))
+
+        # check GetLayerStack behavior
+        self.assertEqual(stage.GetLayerStack()[0], stage.GetSessionLayer())
+        self.assertEqual(stage.GetEditTarget().GetLayer(), stage.GetSessionLayer())
+
+        # set the edit target to the root layer
+        stage.SetEditTarget(stage.GetRootLayer())
+        self.assertEqual(stage.GetEditTarget().GetLayer(), stage.GetRootLayer())
```

Comment:
@ppt-adsk We now need to make sure we are in the correct target layer before making any edits.  

Also please note that this logic would soon change once #661 is in. I will refactor this chunk of code since the same logics is repeated for many of our test cases.